gcloud/orb.yaml

```diff
@@ -483,5 +483,56 @@ jobs:
           image: <<parameters.project>>/<<parameters.image>>
           registry: <<parameters.registry>>
 
+  artifacts-package-publish:
+    description: >
+      This job will publish an already built package to a GCP package
+      repository.
+    docker:
+      - image: <<parameters.executor>>
+    resource_class: <<parameters.resource_class>>
+    parameters:
+      creds:
+        default: GCLOUD_SERVICE_KEY
+        description: >
+          Name of environment variable storing the base64-encoded service key
+          for the GCP project.
+        type: env_var_name
+      executor:
+        default: python:3.7.10
+        description: >
+          Name of the python image to use to execute the job.
+        type: string
+      path:
+        default: .
+        description: >
+          Path to the directory containing your packages.
+        type: string
+      project:
+        description: >
+          Name of GCP project to which we will push.
+        type: string
+      resource_class:
+        default: medium
+        type: string
+      repository-url:
+        description: >
+          URL of GCP package repository to which we will push.
+        type: string
+      workspace:
```

Comment:
This one doesn't need to be configurable -- this doesn't define what the workspace must be named but rather how it will be accessed within this job. Rather, Circle works by having a single workspace for all jobs with direct DAG requirements; `attach_workspace` takes "the workspace available to this job" and mounts it at a given location for local processing.